### PR TITLE
[[ Bug ]] Fix colorization of msg box autocomplete

### PR DESCRIPTION
**Pull request by montegoulding**

### CODE REVIEW

File: Toolset/palettes/message box/behaviors/revmessageboxsinglelinemessagebehavior.livecodescript [hunk 1]
```diff
@@ -105,13 +105,16 @@ command revDisplayAutoComplete pString
...
-   get the long id of field "auto complete"
-   send "revSEColorizeField it" to stack "revSEUtilities" in 100 milliseconds
+   send "__ColorizeAutocomplete" to me in 100 milliseconds
    put the result into sColorizeAutocompleteMsg
-   set the textColor of char 1 to -1 of field "auto complete" to "gray70"
    unlock screen
 end revDisplayAutoComplete
 
+on __ColorizeAutocomplete
+   dispatch "revSEColorizeField it" to stack "revSEUtilities" with the long id of field "auto complete"
```
Comment (livecodepanos) [inline Toolset/palettes/message box/behaviors/revmessageboxsinglelinemessagebehavior.livecodescript]: The `it` probably needs to be removed here

Comment (montegoulding) [reply Toolset/palettes/message box/behaviors/revmessageboxsinglelinemessagebehavior.livecodescript]: yes it does... interestingly it still works which is I guess a bug in dispatch

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok 295a5ec

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok 295a5ec0bef3cd70800b8ce5b52b2e4180b7f42e

**Pull request merged**